responder/responder.go

```diff
@@ -39,9 +39,23 @@ func (r *Responder) JSON(ctx context.Context, w http.ResponseWriter, status int,
 	w.WriteHeader(status)
 
 	if _, err = w.Write(b); err != nil {
-		log.Error(ctx, "failed to write response", err, log.Data{
-			"response": string(b),
-		})
+		var logData log.Data
+		bodyLength := len(b)
+		if bodyLength > 1000 {
```

Comment:
Tiny last suggestion. We've used the `1000` value four times, perhaps a constant would be a good approach in case we want to change it easily.